Tools/Schema.NET.Tool/Schema.NET.Tool.csproj

```diff
@@ -1,15 +1,27 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Build">
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Does it matter what target framework this is? Can't we just use the latest `net5.0`?

Reply:
We have to use .NET Standard 2.0 because the generator runs in the frameworks for the target that uses the generator. Because we target .NET Framework and older .NET Core versions, we need use the most compatible target framework for the generator.